static/app/views/app/index.tsx

```diff
@@ -57,18 +58,23 @@ function App({children, params}: Props) {
   const api = useApi();
   const user = useUser();
   const config = useLegacyStore(ConfigStore);
+  const {visible: isModalOpen} = useGlobalModal();
   const preloadData = shouldPreloadData(config);
 
   // Command palette global-shortcut
   const commandPaletteHotkeys = useMemo(() => {
-    return {
-      match: ['command+shift+p', 'command+k', 'ctrl+shift+p', 'ctrl+k'],
-      includeInputs: true,
```

Comment:
I'm not a big command-palette user but this might be pretty annoying to relearn, so the other option was to just move `ctrl+k` out into an entry without `includeInputs`, so all the other commands still work as expected in text inputs. Having one command act differently felt a little weird too though.